src/decorators.ts

```diff
@@ -793,6 +783,17 @@ export function getAllRegisteredClassNames() {
   return cnames;
 }
 
```

Comment:
Can we also remove `classesByCtor` in this PR. as far as I can tell, we never put anything in that map

Reply:
Can we remove it until we add that aspiration?